services/web/server/src/simcore_service_webserver/users/_api.py

```diff
@@ -0,0 +1,90 @@
+import logging
+from typing import NamedTuple
+
+from aiohttp import web
+from models_library.emails import LowerCaseEmailStr
+from models_library.users import UserID
+from pydantic import EmailStr, PositiveInt, parse_obj_as
+from simcore_postgres_database.models.users import (
+    FullNameTuple,
+    UserNameConverter,
+    UserStatus,
+)
+
+from ..db.plugin import get_database_engine
+from ..email.utils import send_email_from_template
+from ..products.api import get_current_product, get_product_template_path
+from ._db import get_user_or_raise
+from ._db import list_user_permissions as db_list_of_permissions
+from ._db import update_user_status
+from .schemas import Permission
+
+_logger = logging.getLogger(__name__)
+
+
+async def list_user_permissions(
+    app: web.Application, user_id: UserID, product_name: str
+) -> list[Permission]:
+    permissions: list[Permission] = await db_list_of_permissions(
+        app, user_id=user_id, product_name=product_name
+    )
+    return permissions
+
+
+class UserCredintialsTuple(NamedTuple):
```

Comment:
```suggestion
class UserCredentialsTuple(NamedTuple):
```

Reply:
No, no... I really menat that. it's a new word ;-)